src/main/java/turbo/funicular/web/UiController.java

```diff
@@ -56,6 +56,16 @@ public HttpResponse home(Authentication authentication) {
         return HttpResponse.ok(model(authentication));
     }
 
+    @Get("/ghUser/{ghId}")
+    @View("featured_user")
```

Comment:
```suggestion
    @View("profile")
```

Incluso `user` podría funcionar